DependencyInjection/Configuration.php

```diff
@@ -42,7 +42,12 @@ public function getConfigTreeBuilder()
                 ->arrayNode('access_denied_listener')
                     ->canBeEnabled()
                     ->beforeNormalization()
-                        ->ifArray()->then(function($v) { if (!empty($v) && empty($v['formats'])) { unset($v['enabled']); $v = array('enabled' => true, 'formats' => $v); } return $v; })
+                        ->ifArray()->then(function ($v) { if (!empty($v) && empty($v['formats'])) {
+     unset($v['enabled']);
```

Comment:
-- Please reply above this line --

-----------------------------------------------------------
## Graham replied, on Sep 15 @ 12:45pm (UTC):

Hi there,

This is definitely an issue with php-cs-fixer. Please create an issue
on their repo.

Cheers,

Graham Campbell
support@alt-three.com


-----------------------------------------------------------
## Friendsofsymfony/fosrestbundle sent a message, on Sep 15 @ 12:44pm (UTC):

In DependencyInjection/Configuration.php:

> @@ -42,7 +42,12 @@ public function getConfigTreeBuilder()
>                  ->arrayNode('access_denied_listener')
>                      ->canBeEnabled()
>                      ->beforeNormalization()
> -                        ->ifArray()->then(function($v)
{ if (!empty($v) && empty($v['formats'])) { unset($v['enabled']); $v =
array('enabled' => true, 'formats' => $v); } return $v; })
> +                        ->ifArray()->then(function ($v)
{ if (!empty($v) && empty($v['formats'])) {
> +     unset($v['enabled']);

Yeh, it's a bug with php-cs-fixer.
-
Reply to this email directly or view it on GitHub.

-----------------------------------------------------------
Powered by Help Scout:
http://www.helpscout.net/l/main/?utm_campaign=word-of-mouth&utm_medium=email&utm_source=wom&utm_term=Alt%20Three%20Services%20Limited